.github/workflows/compliance.yml

```
@@ -0,0 +1,30 @@
+          meson setup meson-debug --buildtype debugoptimized
+          meson compile -C meson-debug
+      - name: Emit placeholder compliance report
+        run: |
+          cat > compliance.json <<'JSON'
+          {"implementation":"libgitledger","version":"0.0.0","date":"1970-01-01T00:00:00Z","results":[],"summary":{"core":"PARTIAL","policy":"N/A","wasm":"N/A"}}
+          JSON
+      - uses: actions/upload-artifact@v4
+        with:
+          name: compliance.json
+          path: compliance.json
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)